uPortal-api/uPortal-api-platform-impl/src/main/java/org/apereo/portal/api/portlet/DelegateState.java

```diff
@@ -32,6 +33,9 @@
      *     used
      */
     public DelegateState(PortletMode portletMode, WindowState windowState) {
+        Assert.notNull(portletMode, "Argument 'portletMode' cannot be null");
```

Comment:
:question: I haven't seen `Assert` used much in uPortal source, what are the impacts of this vs a null guard conditional?
:question: @apetro @vertein @drewwills @bjagg Is usage of `Assert` something that has been discussed before?